### DISCUSSION TITLE
Add target branch to PR title + Dashboard

### DISCUSSION THREAD

**Discussion by georgkrause** [renovatebot/renovate#13706] [Request Help]
Hello,

its me again, with another question. I am not sure if its supported, but it was there and stopped working yesterday. We use renovate to update dependencies in different branches and I would love to see the target branch of the PR/MR to be part of the title of the PR and to be visible in the dependency dashboard. But I don't want this to be part of the commit message. Is this possible? As far as I can tell, the config `prTitle` is deprecated and shouldn't be used anymore, is there an alternative?

Thank you all!

**Comment by rarkins** [comment]
I thought the baseBranch should be automatically added to the title if you have multiple of them. Please create a public reproduction we can take a look at

**Reply by georgkrause** [reply]
Actually the project I am working on is public, its maybe not a minimal way to reproduce this. 

The config is here: https://dev.funkwhale.audio/funkwhale/funkwhale/-/blob/develop/.gitlab/renovate.json
the dependency dashboard is here: https://dev.funkwhale.audio/funkwhale/funkwhale/-/issues/1679

I think it added the name of the baseBranch until I added those lines to the configuration:

```
{
      "matchDepTypes": ["devDependencies"],
      "matchBaseBranches": ["stable"],
      "enabled": false
    }
```

**Reply by rarkins** [reply]
I found the relevant code: https://github.com/renovatebot/renovate/blob/4f65b57225a42f9deb010f8dbfeb06eb94c0dd69/lib/workers/repository/updates/generate.ts#L221

But there's no other occurrence of this `hasBaseBranches` in our code, so it seems likely it got removed by accident.

@olegkrivtsov I think you broke this in https://github.com/renovatebot/renovate/commit/e148b66d8a7be546084fc2437b9b0f948df956aa

**Reply by rarkins** [reply]
Reverting those changes in #13715

**Reply by viceice** [reply]
Please don't, we only need to readd:

```ts
if (config.baseBranches.length > 1) {
    baseBranchConfig.branchPrefix += `${baseBranch}-`;
    baseBranchConfig.hasBaseBranches = true;
  }
```

You proposed to remove that 😉

**Reply by viceice** [reply]
I've a better fix than revert
- https://github.com/renovatebot/renovate/pull/13716

**Reply by viceice** [reply]
Looks like the code was removed by accident here:
https://github.com/renovatebot/renovate/pull/12514/commits/23b48489a429a288c7ff404859ade2877095a94a#diff-34f2d644001d505d48bb39f08261487e687cabdc839943de43f4369ab974f024L60-L63
